TestFoundation/TestStream.swift

```diff
@@ -7,6 +7,30 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if (os(Linux) || os(Android))
```

Comment:
Please guard this block and tests that use internal members with `#if NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT`. It's not always available.